packages/w3up-client/test/result.test.js

```diff
@@ -0,0 +1,24 @@
+import * as Result from '../src/result.js'
+import assert from 'assert'
+
+describe('Result', () => {
+  it('expect throws on error', async () => {
+    assert.throws(() => Result.expect({ error: new Error('Boom') }), /Boom/)
```

Comment:
I had to go read the src when i saw this in use before to figure out what it's doing.

Maybe `Result.ok` or `Result.assert` would be more readable to me.

`Result.expect` hinted you were doing some kind of expectation about the result, but didn't tell me much about exactly what was being expected.

kinda like how some assertion libraries do `expect(target).toBeTruthy` not just `expect(target)`

Reply:
Sorry I should have added more comments it's basically [Result.expect from Rust](https://doc.rust-lang.org/std/result/enum.Result.html#method.expect).

[`Result.unwrap` is another](https://doc.rust-lang.org/std/result/enum.Result.html#method.unwrap) if you like that name better, but it does not take second param to throw.